packages/editor/src/components/inserter-with-shortcuts/style.scss

```diff
@@ -1,5 +1,5 @@
 .editor-inserter-with-shortcuts {
-	display: flex;
+	//display: flex;
```

Comment:
Is that intended?

Reply:
Raaargh no, sorry. One sec fixing.

Reply:
Sorry about that. Fixed not only this typo, but another one. 

<img width="687" alt="screen shot 2018-08-21 at 10 00 35" src="https://user-images.githubusercontent.com/1204802/44388765-38ccfb00-a529-11e8-94bf-187b2436fcc0.png">

<img width="331" alt="screen shot 2018-08-21 at 10 01 11" src="https://user-images.githubusercontent.com/1204802/44388768-3a96be80-a529-11e8-9759-1c7e2a29cb39.png">